roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1007,6 +1007,41 @@ def get_local_facts_from_file(filename):
 
     return local_facts
 
+def set_proxy_facts(facts):
```

Comment:
These values aren't going to be persisted... Also, we should probably configure this as an additive fact, so that we don't accidentally remove entries that are set.

Since we are only relying on common facts for the computed values, we should be able to set all of these values within init_local_facts so that we can ensure that the values are persisted.